styles/blocks/f-standard-text.css

```diff
@@ -0,0 +1,75 @@
+/*
+standard heading and text for bigger text blogs.
+
+## EXAMPLE 1 
+```
+<div class="kalei-background-white"> <!-- only here for the style guide - should not be used on the website -->
+
+<h2 class="f-text__heading">Heading explaining what the text will be about</h2>
+
+<p class="f-text__text">
+    Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+    Nulla perspiciatis, atque id, tenetur excepturi et repellendus dolorem 
+    necessitatibus corporis fuga? Dolorem quasi earum omnis nam obcaecati 
+    autem doloremque voluptate fugiat.
+    Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+    Nulla perspiciatis, atque id, tenetur excepturi et repellendus dolorem 
+    necessitatibus corporis fuga? Dolorem quasi earum omnis nam obcaecati 
+    autem doloremque voluptate fugiat.
+    Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+    Nulla perspiciatis, atque id, tenetur excepturi et repellendus dolorem 
+    necessitatibus corporis fuga? Dolorem quasi earum omnis nam obcaecati 
+    autem doloremque voluptate fugiat.
+    Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+    Nulla perspiciatis, atque id, tenetur excepturi et repellendus dolorem 
+    necessitatibus corporis fuga? Dolorem quasi earum omnis nam obcaecati 
+    autem doloremque voluptate fugiat.
+</p>
+
+<p class="f-text__text">
+    Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+    Nulla perspiciatis, atque id, tenetur excepturi et repellendus dolorem 
+    necessitatibus corporis fuga? Dolorem quasi earum omnis nam obcaecati 
+    autem doloremque voluptate fugiat.
+    Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+    Nulla perspiciatis, atque id, tenetur excepturi et repellendus dolorem 
+    necessitatibus corporis fuga? Dolorem quasi earum omnis nam obcaecati 
+    autem doloremque voluptate fugiat.
+    Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+    Nulla perspiciatis, atque id, tenetur excepturi et repellendus dolorem 
+    necessitatibus corporis fuga? Dolorem quasi earum omnis nam obcaecati 
+    autem doloremque voluptate fugiat.
+    Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+    Nulla perspiciatis, atque id, tenetur excepturi et repellendus dolorem 
+    necessitatibus corporis fuga? Dolorem quasi earum omnis nam obcaecati 
+    autem doloremque voluptate fugiat.
+</p>
+</div>
+```
+
+
+
+*/
+
+@import "../variables";
+/*  the BEM block class is missing -> .f-text {} */
+
+.f-text__heading {
+    margin-top: 0rem;
+    margin-bottom: 0.65rem;
+
+    color: var(--color-grey-dark);
```

Comment:
Remember to separate CSS blocks according to [the comment in master.css](https://github.com/Firefund/styleguide/blob/master/styles/master.css).
I'll put it in the README as well.